script/test_Dockerfile

```diff
@@ -16,16 +16,24 @@ RUN apt-get update && apt-get install -y \
     python-minimal \
     --no-install-recommends
 
+# install bats 
+RUN cd /tmp \
```

Comment:
We should specify an exact commit or version in the repo to checkout. This means that we don't have to worry about inconsistencies in the test framework.